myst/src/utils/config.rs

```diff
@@ -103,8 +110,14 @@ impl Config {
             aws_secret: config.get_str("aws_secret").unwrap().to_string(),
             temp_data_path: config.get_str("temp_data_path").unwrap().to_string(),
             download_frequency: config.get_int("download_frequency").unwrap() as u64,
-            plugin_path: config.get_str("plugin_path").unwrap(),
+            plugin_path: config.get_str("plugin_path").unwrap_or(String::from("/usr/share/myst/plugins/metrics-reporter")),
             ssl_for_metrics: config.get_bool("ssl_for_metrics").unwrap(),
+            rollup_size: config.get_int("rollup_size").unwrap_or(7*24*60*60) as u32,
+            num_containers: config.get_int("num_containers").unwrap_or(5) as usize,
+            container_id: config.get_int("container_id").unwrap_or(1) as usize,
```

Comment:
num_containers and container_id will need to come from a separate file as these are dynamically injected.
Dynamic injection in k8s works via volumes.
The file name can be /etc/myst/shard-config/shard-config.toml or anything similar. We will mount the  /etc/myst/shard-config dir from outside.